website_snippet_mass_mailing_partner/views/snippets.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- © 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<openerp>
+<data>
+
+<template id="mailing_list_subscribe"
+          inherit_id="mass_mailing.mailing_list_subscribe">
+    <xpath expr="//input[@class='js_subscribe_email form-control']"
+           position="before">
+        <input
+            name="name"
+            class="js_subscribe_name form-control"
+            placeholder="your name..."/>
```

Comment:
Yes, but you must be able to select the name granularity by snippet. In some occasions, you want the snippet to serve only for logued users, in other, for all introducing the name. I have another doubt now. If you are logged, the name of the user is respected?